### PR TITLE
[#2424] improvement(spark-connector): make all Spark SQL keywords uppercase for consistency

### PR DESCRIPTION
**Pull request by ichuniq**
### What changes were proposed in this pull request?

Make all Spark SQL keywords used in spark connector integration test uppercase

### Why are the changes needed?

Fix: #2424 

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
`./gradlew build -PskipTests`

### CODE REVIEW

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java [hunk 1]
```diff
@@ -63,11 +63,11 @@ void testDropSchema() {
...
 
-    sql("create database " + testDatabaseName);
+    sql("CREATE DATABASE " + testDatabaseName);
     databases = getDatabases();
     Assertions.assertTrue(databases.contains(testDatabaseName));
 
-    sql("drop database " + testDatabaseName);
+    sql("DROP DATABASE " + testDatabaseName);
     databases = getDatabases();
     Assertions.assertFalse(databases.contains(testDatabaseName));
```
Comment (FANNG1) [inline integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java:73]: please change this too
```
sql("drop database notExists")
```

Comment (ichuniq) [reply integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java:73]: Just add it!

**Comment by FANNG1** [comment]
LGTM, let's wait CI to finish.

**Review by FANNG1 — APPROVED** [review]

**Comment by FANNG1** [comment]
@ichuniq merged to main, welcome to join the Gravitino community, and hope to enjoy the journey!

**Pull request merged**